remove old dependency for logrus

We get build failures, because of your dependency

You should update github.com/Sirupsen/logrus to github.com/sirupsen/logrus

    % make test
    go test -v -race github.com/szuecs/external-dns github.com/szuecs/external-dns/controller github.com/szuecs/external-dns/endpoint github.com/szuecs/external-dns/internal/testutils  github.com/szuecs/external-dns/pkg/apis/externaldns github.com/szuecs/external-dns/pkg /apis/externaldns/validation github.com/szuecs/external-dns/plan github.com/szuecs/external- dns/provider github.com/szuecs/external-dns/registry github.com/szuecs/external-dns/source
    ../../kubernetes-incubator/external-dns/vendor/github.com/infobloxopen/infoblox-go-client/lock.go:5:2: case-insensitive import collision: "github.com/kubernetes-incubator/external-dns/vendor/github.com/Sirupsen/logrus" and "github.com/kubernetes-incubator/external-dns/vendor/github.com/sirupsen/logrus"
    vendor/github.com/infobloxopen/infoblox-go-client/lock.go:5:2: case-insensitive import collision: "github.com/szuecs/external-dns/vendor/github.com/Sirupsen/logrus" and "github.com/szuecs/external-dns/vendor/github.com/sirupsen/logrus"
    make: *** [test] Error 1

--- a/lock.go
+++ b/lock.go
@@ -2,7 +2,7 @@ package ibclient
 
 import (
 	"fmt"
-	"github.com/Sirupsen/logrus"
+	"github.com/sirupsen/logrus"
 	"math/rand"
 	"time"
 )

